setup.sh

```diff
@@ -17,21 +17,30 @@ else
    SHERLOCK=false
 fi
 
-echo "Next, please supply the domain name of the resource. If you are using farmshare or sherlock, leave as default."
-echo
-read -p "Domain Name for Resource (default: rice.stanford.edu for farmshare) > " DOMAINNAME
-DOMAINNAME=${DOMAINNAME:-rice.stanford.edu}
-
-echo "Next, please supply the prefix of the compute nodes that are used in your cluster resource. We will use this to check for assignment of 
-compute node when we submit the sbatch script. If you are using sherlock or farmshare (without gpu capability), then prefixes are set for you."
-echo
-read -p "Compute Node Prefix identifier (default: sh for sherlock) > "  MACHINEPREFIX 
-
-if $SHERLOCK 
+if $SHERLOCK
 then
    MACHINEPREFIX=${MACHINEPREFIX:-sh}
+   USE_LSOF=${USE_LSOF:-/usr/sbin/lsof}
+   DOMAINNAME=${DOMAINNAME:-login.sherlock.stanford.edu}
 else
```

Comment:
this needs an else if to see if we are on farmshare, and if not, ask the user to enter all these variables.

Reply:
Oh I see you did it below! So bash has else if statements - I think you want to use one above and add farmshare as an `elif` and then the `else` should be the content below that asks the user to prompt.

Reply:
I tried with the elif - and for some reason bash is not responding nicely. So i had to go the more inelegant option.

Reply:
Let's try to get this right :)

```bash
RESOURCE=sherlock
if [[ "${RESOURCE}" == "sherlock" ]]; then
    echo "The resource is sherlock"
elif [[ "${RESOURCE}" == "farmshare" ]]; then
    echo "The resource is farmshare"
else
    echo "The resource is neither."
fi
```